spec/features/mediated_deposit_spec.rb

```diff
@@ -0,0 +1,52 @@
+# typed: false
```

Comment:
Since I was already writing a mediated deposit feature spec to test the versioning/state bugfix, I thought I'd toss a basic mediated deposit feature spec in too.